scipy/signal/__init__.py

```diff
@@ -343,14 +345,19 @@
                       'nuttall', 'blackmanharris', 'flattop', 'bartlett',
                       'barthann', 'hamming', 'kaiser', 'gaussian',
                       'general_gaussian', 'chebwin', 'cosine',
-                      'hann', 'exponential', 'tukey')
+                      'hann', 'exponential', 'tukey', 'taylor', 'lanczos',
+                      'dpss')
```

Comment:
These were missing as highlighted by the test

Reply:
Well... IIUC, these were only ever added in `scipy.signal.window`, and not exposed through `scipy.signal`, which is a good thing. While I don't think it wouldn't be a huge issue to add them here for simplicity of our deprecation mechanism, the better approach IMO is to skip the deprecation test for those three.

Reply:
Ha yes, you're right, didn't consider that possibility. Have reverted the change.